Jenkinsfile.d/package

```diff
@@ -41,8 +41,8 @@ pipeline {
     AZURE_VAULT_TENANT_ID     = credentials('azure-vault-tenant-id')
     GPG_FILE                  = 'jenkins-release.gpg'
     GPG_PASSPHRASE            = credentials('release-gpg-passphrase')
-    PACKAGING_GIT_REPOSITORY  = 'git@github.com:jenkinsci/packaging.git'
-    PACKAGING_GIT_BRANCH      = 'INFRA-910-core-release-automation'
+    PACKAGING_GIT_REPOSITORY  = 'git@github.com:olblak/packaging.git'
+    PACKAGING_GIT_BRANCH      = 'master'
```

Comment:
windows seems outdated on this branch, there's newer commits on jenkinsci/packaging

Reply:
Yep I fixed my branch and go back on jenkinsci/packaging